tron/serialize/runstate/statemanager.py

```diff
@@ -60,7 +59,8 @@ class StateMetadata(object):
     RunState interface as Jobs and Services.
     """
     name = 'StateMetadata'
-    version = tron.__version_info__
+    # version = tron.__version_info__
+    version = (0, 7, 0, 0)
```

Comment:
Does the 7 matter? Why have the tuple if you're only planning on using the first number?
Or is this supposed to be related to the Tron version number in some way?

Reply:
making it not a tuple will break compatibility of state metadata with our 0.7.x releases. 7 doesn't really matter.

Reply:
Ah, right, rollback safety... Got it. Is there a way we can make it more obvious what this version means and when we should bump it?